src/main/java/sparksoniq/spark/iterator/function/UrlValidator.java

```diff
@@ -0,0 +1,34 @@
+package sparksoniq.spark.iterator.function;
+
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.spark.api.java.JavaSparkContext;
+import sparksoniq.exceptions.OurBadException;
+import sparksoniq.exceptions.SparksoniqRuntimeException;
+import sparksoniq.spark.SparkSessionManager;
+
+import java.io.File;
+import java.io.IOException;
+
+public class UrlValidator {
+
+    public static boolean isValid(String url) {
+        if (url.startsWith("file://") || url.startsWith("/") || url.startsWith("./")) {
+            File f = new File(url);
+            return f.exists();
+        }
```

Comment:
@ghislainfourny what is our current status regarding workspace definition and using file names without any prefix such as ./ or // etc.  ie. referring as "file-in-workspace.txt". This may require an extra touch here